src/foundry/foundry.js/collections/documentCollections/worldCollections/scenes.d.ts

```diff
@@ -0,0 +1,62 @@
+import { ConfiguredDocumentClass } from '../../../../../types/helperTypes';
+import { BaseScene } from '../../../../common/documents.mjs';
+
+declare global {
+  /**
+   * The singleton collection of Scene documents which exist within the active World.
+   * This Collection is accessible within the Game object as game.scenes.
+   *
+   * @see {@link Scene} The Scene entity
+   * @see {@link SceneDirectory} The SceneDirectory sidebar directory
+   */
+  class Scenes extends WorldCollection<typeof foundry.documents.BaseScene, 'Scenes'> {
+    /** @override */
+    static documentName: 'Scene';
+
+    /**
+     * Return a reference to the Scene which is currently active
+     */
+    get active(): Scene | undefined;
+
+    /**
+     * Return the current Scene target.
+     * This is the viewed scene if the canvas is active, otherwise it is the currently active scene.
+     */
+    get current(): Scene | undefined;
+
+    /**
+     * Return a reference to the Scene which is currently viewed
+     */
+    get viewed(): Scene | undefined;
+
+    /**
+     * Handle pre-loading the art assets for a Scene
+     *
+     * @param sceneId - The Scene id to begin loading
+     * @param push    - Trigger other connected clients to also pre-load Scene resources
+     *                  (default: `false`)
+     */
+    preload(sceneId: string, push?: boolean): SocketIOClient.Socket | Promise<unknown[]>;
+    // TODO: This can be typed better
```

Comment:
Not sure if it's worth to type this better. I'm pretty sure the return type is not intended to be used at all. The `Promise` may contain a `Sound` as first element, but it doesn't have to. The last element will always be of type `void`, i.e. most likely its `undefined`, but it might be `null`, too. Not really useful... but if you really want to do it correctly, it would be
```ts
     preload(sceneId: string, push?: boolean): SocketIOClient.Socket | Promise<[Sound, void] | [void]>;
 ```
 `Sound` hasn't been typed yet, though, it's a new top-level class (see #889). You could add it as a dummy, I suppose.

Reply:
Slept about it and while it is opinionated, I think it is better if it is unknown and no one can use this type.